drivers/usb/udc/Kconfig.mcux

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config USB_UDC_NXP_EHCI
```

Comment:
Prefix must be UDC_, not USB_UDC. Please fix it for all new Kconfig options.
config UDC_NXP_EHCI

Reply:
Updated. Hi @dleach02 Because this change, I create one hal_nxp pr https://github.com/zephyrproject-rtos/hal_nxp/pull/400

Reply:
Can you do
```
  include_ifdef(CONFIG_DT_HAS_NXP_USBPHY_ENABLED  middleware_usb_phy)
  include_ifdef(CONFIG_UDC_NXP_EHCI               middleware_usb_device_ehci)
  include_ifdef(CONFIG_UDC_NXP_IP3511             middleware_usb_device_ip3511fs)
```
in `modules/hal_nxp/CMakeLists.txt`?

Reply:
> Updated. Hi @dleach02 Because this change, I create one hal_nxp pr [zephyrproject-rtos/hal_nxp#400](https://github.com/zephyrproject-rtos/hal_nxp/pull/400)

Thanks